packages/table/src/column.tsx

```diff
@@ -21,6 +30,13 @@ export interface IColumnProps extends IColumnNameProps, IProps {
     id?: string | number;
 
     /**
+     * A single value or set of `ColumnLoading` options. If non-null, loading
+     * state rendering will take precendence over `renderCell` and
+     * `renderColumnHeader` callbacks.
+     */
+    loadingOptions?: Set<ColumnLoadingOption>;
```

Comment:
I like that Sets imply uniqueness, but I'd rather not use this type. We should just use an plain arrays instead, like the SelectionMode option in `<Table>`.